### PR TITLE
Terms view

### PR DESCRIPTION
**Pull request by AxelJNS**
Created a more user friendly user interface for the terms page. Added button layouts for adding, deleting, and comparison functionalities

### CODE REVIEW

File: views/pages/courses/index.ejs [hunk 1]
```diff
@@ -1,87 +1,55 @@
...
+<% currentCourses = [{ number: "MATH 158", name: "Calculus 3", credit: 4, more: { grade: 'A',
+category: ['Mathematical Course', 'Mobile Applications Course'], date: 'Current', } },{ number: "CS
+404", name: "Adv Web and Internet", credit: 3, more: { grade: 'B+', category: ['Mobile Applications
+Course'], date: 'Current', } },{ number: "MSCS 234", name: "Machine Learning", credit: 3, more: {
+grade: 'B', category: ['Mathematical Course', 'Mobile Applications Course'], date: 'Current', } }]
+completedCourses = [{ number: "MATH 157", name: "Calculus 2", credit: 4, more: { grade: 'A-',
+category: ['Mathematical Course'], date: 'Fall 2020', } },{ number: "CS 380", name: "Web and
+Internet", credit: 3, more: { grade: 'A', category: ['Mathematical Course'], date: 'Spring 2018', }
+},{ number: "ENGL 102", name: "English 2", credit: 2, more: { grade: 'C+', category: ['Communication
+Skills'], date: 'Summer 2019', } }] %>
```
Comment (Pirundy) [inline views/pages/courses/index.ejs]: This formatting will break the array

**Review by Pirundy — CHANGES_REQUESTED** [review]
Reformatting array in course page will break array if I am correct

File: views/pages/courses/index.ejs (hunk 1 above)
Comment (AxelJNS) [reply views/pages/courses/index.ejs]: I did not chage anything in this file. My OCD just made me save this by habit. All changes from my branch in this file can be deleted

Comment (AxelJNS) [reply views/pages/courses/index.ejs]: I can change it back real quick

**Review by Pirundy — APPROVED** [review]
Looks better thanks!

File: views/pages/courses/index.ejs (hunk 1 above)
Comment (Pirundy) [reply views/pages/courses/index.ejs]: Yeah I found out the fun way that the prettier formatting breaks the array each time I would format it lol

**Review by Jake-Derrick — APPROVED** [review]

**Comment by Pirundy** [comment]
I think I fixed the merge issues please double check to make sure all your work looks correct now @AxelJNS

**Pull request merged**